docs/reference/commands.rst

```diff
@@ -16,7 +16,6 @@ inmanta
    :func: cmd_parser
    :prog: inmanta
 
-inmanta-cli
-###########
-
-Use ``inmanta-cli --help``
+.. click:: inmanta.main:cmd
```

Comment:
The inmanta-cli title is missing. Now the layout of the page looks as follows:

![doc-layout](https://user-images.githubusercontent.com/2684622/80070640-8fcb7480-8543-11ea-822e-57308d81bef6.png)

I would expect something like:
- command reference:
-- inmanta
-- inmanta-cli